webapp/loggedin/appSideBar/appSideBarFooter.js

```diff
@@ -1,31 +1,42 @@
 import React from 'react'
 
-const AppSideBarFooter = ({logout, opened}) => (
-  <div className="app-sidebar__footer">
-    <a className="btn btn-s btn-of-light-xs"
-       onClick={() => logout()}
-       style={{
-         display: 'flex',
-         alignItems: 'baseline',
-       }}>
-            <span className={`icon icon-exit ${opened ? ' icon-left' : ''}`}
-                  style={{transform: 'scaleX(-1)'}}/>
-      {
-        opened
-          ? <span>Logout</span>
-          : null
-      }
-    </a>
+import AppContext from '../../app/appContext'
 
-    <a className="btn btn-of-link app-sidebar__btn-of"
-       href="http://www.openforis.org"
-       target="_blank">
-      {
-        opened
-          ? 'Open Foris' : 'OF'
-      }
-    </a>
-  </div>
-)
+class AppSideBarFooter extends React.PureComponent {
+  render () {
+    const { logout, opened } = this.props
+    const { t } = this.context.i18n
+
```

Comment:
It's not clear when reading just t in the code.
What if we import the ```const i18n = this.context.i18n``` and then use it as```i18n.t('key')```?

Reply:
ok